lib/Doctrine/Inflector/Inflector.php

```diff
@@ -37,7 +37,7 @@ public function __construct(WordInflector $singularizer, WordInflector $pluraliz
      */
     public function tableize(string $word) : string
     {
-        return mb_strtolower(preg_replace('~(?<=\\w)([A-Z])~u', '_$1', $word));
+        return mb_strtolower((string) preg_replace('~(?<=\\w)([A-Z])~u', '_$1', $word));
```

Comment:
These changes are not valid, it hides the error instead of reacting on it.

Reply:
https://github.com/doctrine/inflector/issues/115

Reply:
I started to throw exceptions when preg_replace returns null but it got so complicated so quick in some areas where we chain multiple preg_replace calls together. I'll work on it in #115

Reply:
Also, do you know in what scenarios preg_replace would return null?

Reply:
https://github.com/phpstan/phpstan/pull/1399#issuecomment-441074705

Reply:
There is also https://github.com/thecodingmachine/safe